cmd/ci-secret-bootstrap/main.go

```diff
@@ -193,8 +207,10 @@ func constructSecrets(config []secretConfig, bwClient bitwarden.Client) (map[str
 			var err error
 			if bwContext.Field != "" {
 				value, err = bwClient.GetFieldOnItem(bwContext.BWItem, bwContext.Field)
-			} else {
+			} else if bwContext.Attachment != "" {
 				value, err = bwClient.GetAttachmentOnItem(bwContext.BWItem, bwContext.Attachment)
+			} else {
+				value, err = bwClient.GetPassword(bwContext.BWItem)
```

Comment:
This should be a switch/case on the value of `Attribute`

Reply:
Done. Also applied to the code in `validate` function.